docs/samples/browser-plugin-meetings/app.js

```diff
@@ -350,7 +359,85 @@ function createMeeting(e) {
   return false;
 }
 
-function joinMeeting({withMedia, withDevice} = {withMedia: false, withDevice: false}) {
+function refreshCaptcha() {
+  const meeting = webex.meetings.getAllMeetings()[selectedMeetingId];
+  meeting.refreshCaptcha()
+    .then(() => {
+      console.log('MeetingsManagement#refreshCaptcha() :: successfully refreshed captcha');
+      meetingsJoinCaptchaImgElm.src = meeting.requiredCaptcha.verificationImageURL;
+      meetingsJoinCaptchaImgElm.hidden = false;
+      meetingsJoinCaptchaElm.type = "text";
+      meetingsJoinCaptchaElm.value = "";
+      refreshCaptchaElm.hidden = false;
+    })
+    .catch((error) => {
+      console.error('MeetingsManagement#refreshCaptcha() :: error refreshing captcha', error);
+      throw (error);
+    });
+}
+
+meetingsListElm.onclick = (e) => {
+  selectedMeetingId = e.target.value;
+  const meeting = webex.meetings.getAllMeetings()[selectedMeetingId];
+
+  if(meeting && meeting.passwordStatus === 'REQUIRED'){
```

Comment:
just wondering... maybe we should add || passwordStatus  === 'UNKNOWN' here too?

Reply:
Sure @marcin-bazyl I will add that..

Reply:
We will update the condition with 
`if(meeting && meeting.passwordStatus === 'REQUIRED' && meeting.passwordStatus !== 'UNKNOWN'){`
I hope this will be fine, will commit after testing.